src/net/sourceforge/kolmafia/request/CrimboCafeRequest.java

```diff
@@ -13,22 +12,22 @@ public class CrimboCafeRequest extends CafeRequest {
   public static final Object[][] MENU_DATA = {
     // Item, itemID, price
     {
-      "Peppermint Nutrition Block", IntegerPool.get(-104), IntegerPool.get(50),
```

Comment:
A topic for another PR, maybe, but do we want to put pseudo-itemids into ItemPool?

Reply:
I think it is a good idea to avoid magic numbers.

Reply:
Having actually started it... they're used in exactly one place, it's reasonably clear what they are (and should be more so after we stop using `Object[][]`) so I'm less keen.